spring-cloud-dataflow-rest-resource/pom.xml

```diff
@@ -41,10 +84,10 @@
 			<groupId>org.springframework.batch</groupId>
 			<artifactId>spring-batch-core</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-task-core</artifactId>
-		</dependency>
+<!--		<dependency>-->
```

Comment:
Do you mind adding a comment (in code) as to why this is commented out?

Reply:
move to separate PR